src/PlaywrightSharp.Tests/BaseTests/PlaywrightSharpBrowserContextBaseTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PlaywrightSharp.Tests.BaseTests
+{
+    /// <summary>
+    /// Based on <see cref="PlaywrightSharpBrowserBaseTest"/>, this calss creates a new <see cref="IBrowserContext"/>
+    /// </summary>
+    public class PlaywrightSharpBrowserContextBaseTest : PlaywrightSharpBrowserBaseTest
+    {
+        internal PlaywrightSharpBrowserContextBaseTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        internal IBrowserContext Context { get; set; }
```

Comment:
this could be `protected`

Reply:
`PlaywrightSharpBrowserContextBaseTest` needs to be public so the tests can be public.
I want the tests to be public so we enforce XML documentation. 
`protected` enforces XML documentation, whereas `internal` doesn't.
Just to make things easier (let's remember we are in the tests library),  this will work and I don't see any downside with the 'internal' keyword.